README.md

```diff
@@ -11,10 +11,20 @@ npm i react-turnstile
 ## Usage
 
 ```jsx
-import Turnstile from "react-turnstile";
+import Turnstile, { useTurnstile } from "react-turnstile";
 
 // ...
 
+// Programmatic access example:
+const turnstile = useTurnstile();
+
+const result = await fetch('/path/to/some/api');
+if (!result.ok) {
+  throw new Error(`Request failed with code ${result.status}`);
+  turnstile.reset();
+}
+
+
```

Comment:
Nonsensical addition, you are using a hook outside of a component.
The example on main also already covers this.